GVRf/Framework/src/org/gearvrf/GVRRenderTexture.java

```diff
@@ -55,6 +55,20 @@ public GVRRenderTexture(GVRContext gvrContext, int width, int height,
     GVRRenderTexture(GVRContext gvrContext, long ptr) {
         super(gvrContext, ptr);
     }
+    
+    /**
+     * Return the width of GVRRenderTexture (FBO)
+     */
+    public int getWidth() {
+    	return NativeRenderTexture.getWidth(getPtr());
+    }
+
```

Comment:
I have just removed the native part of codes and stored width and height in Java part.

We can make similar changes in many other places, i.e. storing useful information in Java side and do not need to spend time calling native methods to retrieve them. This way is in fact the correct way in my opinion, comparing with the current "wired" design of GVRF. :)